opendbc/car/psa/carstate.py

```diff
@@ -0,0 +1,76 @@
+from opendbc.car import structs, Bus
+from opendbc.can.parser import CANParser
+from opendbc.car.common.conversions import Conversions as CV
+from opendbc.car.psa.values import DBC, CarControllerParams
+from opendbc.car.interfaces import CarStateBase
+
+GearShifter = structs.CarState.GearShifter
+TransmissionType = structs.CarParams.TransmissionType
+
+class CarState(CarStateBase):
+  def __init__(self, CP):
+    super().__init__(CP)
+
+  def update(self, can_parsers) -> structs.CarState:
+    cp = can_parsers[Bus.cam]
+    cp_adas = can_parsers[Bus.adas]
+    cp_main = can_parsers[Bus.main]
```

Comment:
```suggestion
    cp_main = can_parsers[Bus.main]
    cp_adas = can_parsers[Bus.adas]
    cp_cam = can_parsers[Bus.cam]
```

then rename the below references